### ISSUE TITLE
New gene (NFU1) for Hpo: Hyperglycinemia HP:0002154 

### ISSUE THREAD

**Issue by lasaaranzasti** [obophenotype/human-phenotype-ontology#8031]
**Preferred term label:**
Hyperglycinemia HP:0002154

**Synonyms**

**Definition (free text, please give PubMed ID)**
- Nizon M, Boutron A, Boddaert N, et al. Leukoencephalopathy with cysts and hyperglycinemia may result from NFU1 deficiency. Mitochondrion. 2014;15:59-64. doi:10.1016/j.mito.2014.01.003
- Ahting U, Mayr JA, Vanlander AV, et al. Clinical, biochemical, and genetic spectrum of seven patients with NFU1 deficiency. Front Genet. 2015;6:123. Published 2015 Apr 13. doi:10.3389/fgene.2015.00123

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
- Hyperglycinemia (OMIM 605711; Oprhanet 401869)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Thanks for pointing this out. I annotated three papers including the Nizon paper that all document Hyperglycinemia. The new annotations will be visible on the website with the next HPO release in about one month.

**Comment by lasaaranzasti** [comment]
Thanks!
Amaia
